### DISCUSSION TITLE
styling the admin...

### DISCUSSION THREAD

**Discussion by skuflow** [oqtane/oqtane.framework#2969] [Q&A]
what is the best way to style the admin area.  If we make a custom theme is there a best practice to also style the admin area so that it looks more like a consistent application?  From what I can tell the admin area just has a single pane.  I want to add a module onto the login page but I can't specify where to put it on the page.  I'd like to have the login on the left and a custom OTP login on the right.

What is the best way to do this and address some of the styling concerns we have with the admin to make the admin more customized for our application?

**Comment by sbwalker** [comment]
Modules are injected into a theme at run-time based on configuration (ie. it is not hard-coded). The default Oqtane theme has many different panes which can be used, but by default the default template only uses the Default Pane for the Admin modules. You can customize any of the Admin pages through a variety of different methods - both manual or automated. With the manual method, login as an administrator, navigate to an Admin page, click the Edit icon at the top left to go into Edit Mode, use the Action menu on modules to move them to different panes, add a Login module from the Control Panel . This is the same approach you would use for manually populating any page in Oqtane with content - the Admin pages are no different.
